README.md

```diff
@@ -47,11 +399,7 @@ Running the app in standalone mode means you will need to install the binaries f
 - [Redis](https://redis.io/)
 - [Hashicorp Vault](https://github.com/hashicorp/vault)
 
-_NB: There is no compatibility with Windows environments at this time._
-
-#### Setup for standalone mode
-
-Make sure you have Postgres, Redis and Vault properly installed & configured. Do _not_ use `make up` since those will start the containers for non-production builds, see [option 1](#option-1---using-docker-only).
+#### Standalone Mode Setup
```

Comment:
The links in this section are all invalid now and need to be revised.

Reply:
To flag up basic Markdown violations, I recommend installing [Markdownlint](https://github.com/DavidAnson/markdownlint) (VSCode).